src/views/payment-sheet-views/paypal-view.js

```diff
@@ -14,24 +16,56 @@ PayPalView.prototype = Object.create(BaseView.prototype);
 PayPalView.prototype.constructor = PayPalView;
 PayPalView.ID = PayPalView.prototype.ID = paymentOptionIDs.paypal;
 
+PayPalView.prototype.setLogLevel = function (paypal) {
+  var level = this.model.merchantConfiguration.paypal.logLevel;
+
+  paypal.setup({
```

Comment:
The reason for this is that it is made so you can't overwrite the setup function.